test/otherFactory.js

```diff
@@ -63,6 +63,16 @@
                     );
                 });
             },
+            formatted: function (callback) {
+                require(["test/DataFactory", "src/other/Table"], function (DataFactory, Table) {
+                    callback(new Table()
+                        .columns(DataFactory.Table.formatted.columns)
+                        .data(DataFactory.Table.formatted.data)
+                        .columnPatterns([".4%", ".6r", "%d-%m-%y"])
```

Comment:
Yes, I need a better sample data set, but if you want to test quickly you can just edit the array in the prop editor